repose-aggregator/core/core-lib/pom.xml

```diff
@@ -137,6 +138,18 @@
 
         <dependency>
             <groupId>org.openrepose</groupId>
+            <artifactId>atom-feed-service-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
if i recall it was done to datastore because of classpathing and deserialization related issues